jujugui/static/gui/src/app/extensions/app-renderer-extension.js

```diff
@@ -58,7 +58,7 @@ YUI.add('app-renderer-extension', function(Y) {
         }
       }
       // If we're in sandbox we don't want to display the switcher.
-      if (this.get('sandbox')) {
+      if (this.get('sandbox') && !this.jem) {
```

Comment:
Drive-by to have the model switcher appear in the sandbox when there are multiple models available.